transaction-view/src/transaction_meta.rs

```diff
@@ -142,6 +142,19 @@ impl TransactionMeta {
             .as_ptr()
             .add(usize::from(self.recent_blockhash_offset)) as *const Hash)
     }
+
+    /// Return an iterator over the instructions in the transaction.
+    /// # Safety
+    /// - This function must be called with the same `bytes` slice that was
+    ///   used to create the `TransactionMeta` instance.
+    pub unsafe fn instructions_iter<'a>(&self, bytes: &'a [u8]) -> InstructionsIterator<'a> {
```

Comment:
Was on the fence about whether this should return the actual type or just an `impl Iterator/ExactSizeIterator`.

Anyone got strong feelings on this one way or another?